site/features.md

```diff
@@ -55,6 +56,20 @@ now re-use the same tools and techniques when deploying applications
 as containers as we would have done when deploying them 'on metal' in
 our data centre.
 
+### <a name="optimised-traffic-transport"></a>Optimised traffic transport
```

Comment:
I would be fine with "Fast data path". It's perhaps not the most descriptive we can come up with, but it's not far off.